modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/ButtonTag.java

```diff
@@ -290,7 +290,7 @@ protected String processCssClasses(Set<String> cssClasses) {
 			cssClasses.add("btn-block");
 		}
 
-		if (Validator.isNotNull(_icon) || _monospaced) {
+		if (_monospaced) {
```

Comment:
Hey @pat270, what do you think about this?

I'm not sure why I added this in the first place... should all Buttons that have an icon `monospaced`? Or all buttons that **only** have an icon? 🤔 

I'm not sure about the specific impact this change might have.. 🤔

Reply:
Regular and monospaced buttons can contain icons. I think this is a good change, but looks like there will be some impact.

- modules/apps/asset/asset-taglib/src/main/resources/META-INF/resources/asset_view_usages/page.jsp
- modules/apps/asset/asset-taglib/src/main/resources/META-INF/resources/select_asset_display_page/page.jsp
- modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/file_entry_upper_tbar.jsp
- modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/info_panel_file_entry.jsp
- modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/info_panel_file_entry.jsp
- modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/image_selector/page.jsp
- modules/apps/item-selector/item-selector-taglib/src/main/resources/META-INF/resources/repository_entry_browser/page.jsp
- modules/apps/journal/journal-web/src/main/resources/META-INF/resources/article/template.jsp
- modules/apps/journal/journal-web/src/main/resources/META-INF/resources/edit_article.jsp
- modules/apps/journal/journal-web/src/main/resources/META-INF/resources/edit_ddm_template.jsp
- modules/apps/layout/layout-taglib/src/main/resources/META-INF/resources/layout_classed_model_usages_view/page.jsp
- modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/search/bar/view.jsp
- modules/apps/product-navigation/product-navigation-global-menu-web/src/main/resources/META-INF/resources/global_menu/global_menu.jsp
- modules/apps/ratings/ratings-taglib/src/main/resources/META-INF/resources/page.jsp


What if we did `Validator.isNotNull(_icon) && Validator.isNotNull(_label) || _monospaced`?

Reply:
> What if we did Validator.isNotNull(_icon) && Validator.isNotNull(_label) || _monospaced?

Do you mean Validator.isNotNull(_icon) && Validator.**isNull**(_label) ?

Reply:
Yes `Validator.isNotNull(_icon) && Validator.isNull(_label)`